src/main/java/io/unlogged/command/ServerMetadata.java

```diff
@@ -11,11 +13,24 @@ public void setAgentServerPort(int agentServerPort) {
     String includePackageName;
     String agentVersion;
     String agentServerUrl;
+	private String hostname;
+	private long createdAt;
+
     public ServerMetadata(String includePackageName, String agentVersion, int agentServerPort) {
         this.includePackageName = includePackageName;
         this.agentVersion = agentVersion;
+		this.createdAt = System.currentTimeMillis() / 1000L;
         this.agentServerPort = agentServerPort;
         this.agentServerUrl = "http://localhost:" + String.valueOf(agentServerPort);
+
+		InetAddress inetAddress;
+		try {
+			inetAddress = InetAddress.getLocalHost();
```

Comment:
This should not be happening here. Parsing logic should be outside pojo.

Reply:
Resolved now. Parsing logic happens before calling the constructor for serverMetadata and the signature of constructor is also changed to include hostname.